xmantissa/test/test_stats.py

```diff
@@ -59,10 +59,9 @@ def test_deliverStatEvents(self):
                 (d['key'], d['value'])
                 for d in parseString(stat['data'])
                 if d['key'] != 'time'])
-        self.assertEqual(
-            data,
-            set([('foo', 'bar'), ('baz', '12'),
-                 ('quux', u'\N{SNOWMAN}'.encode('utf-8'))]))
+        for item in set([('foo', 'bar'), ('baz', '12'),
```

Comment:
The assertion message will tell you which one failed, but I guess the number of items is not prohibitively high, here.